extensions/binding/org.eclipse.smarthome.binding.dmx/src/main/java/org/eclipse/smarthome/binding/dmx/handler/Lib485BridgeHandler.java

```diff
@@ -88,7 +88,7 @@ protected void closeConnection() {
                 try {
                     socket.close();
                 } catch (IOException e) {
-                    logger.warn("Could not close socket {} in {}: {}", receiverNode, this.thing.getUID(), e);
+                    logger.warn("Could not close socket {} in {}: ", receiverNode, this.thing.getUID(), e);
```

Comment:
since this is warn logging provide an `e.getMessage()` instead and re-add the placeholder.